drivers/claims_reporting/app/views/claims_reporting/warehouse_reports/engagement_trends/show.xlsx.axlsx

```diff
@@ -0,0 +1,99 @@
+wb = xlsx_package.workbook
```

Comment:
Whitespace seems wonky on this file -- I guess RuboCop doesn't look a axlsx files.